tests/Api/MauticApiTestCase.php

```diff
@@ -73,6 +145,12 @@ protected function standardTestGetListOfSpecificIds()
             $itemIds[] = $response[$this->api->itemName()]['id'];
         }
 
+        if (is_callable($callback)) {
+            // Allow support to make additional tests based on created associations with this item that
+            // may not have create/edit endpoints
+            $callback($response);
```

Comment:
This should work in 5.3. The only thing is in the method defining $callback, you have to use the `use` statement with variables as it won't recognize `$this` inside the callback. http://sandbox.onlinephpfunctions.com/code/5013ed64742cfd1bc6cc32a571a22d9297f10fc8